halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/model/v1/node/Notification.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2017 URUP.com, Inc.
```

Comment:
Please add urup to the authors as a part of this PR: https://github.com/spinnaker/halyard/blob/master/AUTHORS